packages/template-retail-react-app/app/hooks/use-einstein.js

```diff
@@ -440,51 +429,88 @@ const useEinstein = () => {
         return reco
     }
 
+    const setEventUserParameters = async () => {
```

Comment:
`get` makes more sense in this case. If it's `set`, then the function needs an argument.

```suggestion
    const getEventUserParameters = async () => {
```

Reply:
Applied!